x/market/types/event.go

```diff
@@ -18,6 +18,9 @@ const (
 	evActionLeaseCreated = "lease-created"
 	evActionLeaseClosed  = "lease-closed"
 
+	evOwnerKey       = "owner"
+	evDSeqKey        = "dseq"
+	evGSeqKey        = "gseq"
```

Comment:
can you move these to the test file?  I think it's the only place that they're used.